lib/core/mergeConfig.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var utils = require('../utils');
+
+/**
+ * Config-specific merge-function which creates a new config-object
+ * based on given defaults and instance config.
```

Comment:
Could you please update the comment so it doesn't suggest that this function can be used only to merge defaults with an instance config?